samples/fabric/fabric-cli/src/helpers/helpers.ts

```diff
@@ -228,6 +227,68 @@ const pledgeAsset = async ({
   }
 }
 
+// Used to obtain remote network user certificate from '${networkId}_UsersAndCerts.json' file.
+// This is called during Pledge to get the recipientCert, and during Claim to get the pledgerCert.
+const getUserCertFromFile = (
+  remoteUser: string,
+  remoteNetworkId: string
+) => {
+  const usersAndCertsFile = remoteNetworkId + '_UsersAndCerts.json'
+  const credentialsPath = process.env.MEMBER_CREDENTIAL_FOLDER
+    ? path.resolve(__dirname, process.env.MEMBER_CREDENTIAL_FOLDER)
+    : path.join(__dirname, '../data', 'credentials')
```

Comment:
Can you create a subfolder under `credentials` for these certificate files? To avoid cluttering up the main folder? And add the subfolder name to `.gitignore`.

Reply:
Done.